### PR TITLE
[thci] add setCcmState method in thci

### PR DESCRIPTION
**Pull request by canisLupus1313**
This commit adds API method setCcmState to THCI file.

### CODE REVIEW

**Comment by canisLupus1313** [comment]
@simonlingoogle can I count on Your review today?

**Review by simonlingoogle — APPROVED** [review]
LGTM 👍

File: tools/harness-thci/OpenThread.py [hunk 1]
```diff
@@ -3602,6 +3602,10 @@ def __lstrip0x(s):
 
         return s
 
+    @API
+    def setCcmState(self, state=0):
```
Comment (simonlingoogle) [inline tools/harness-thci/OpenThread.py:3606]:
```suggestion
    assert state in (0, 1), state
    def setCcmState(self, state=0):
```

nit

**Review by jwhui — APPROVED** [review]
Thanks! 👍🏼

**Pull request merged**